scripts/copy_from_pqclean/src/kem/family/kem_family.h

```diff
@@ -9,7 +9,7 @@
 #define OQS_KEM_{{ family }}_{{ scheme['scheme'] }}_length_secret_key {{ scheme['metadata']['length-secret-key'] }}
 #define OQS_KEM_{{ family }}_{{ scheme['scheme'] }}_length_ciphertext {{ scheme['metadata']['length-ciphertext'] }}
 #define OQS_KEM_{{ family }}_{{ scheme['scheme'] }}_length_shared_secret {{ scheme['metadata']['length-shared-secret'] }}
-OQS_KEM *OQS_KEM_{{ family }}_{{ scheme['scheme'] }}_new();
+OQS_KEM *OQS_KEM_{{ family }}_{{ scheme['scheme'] }}_new(void);
```

Comment:
Can you remove the double underscores from `__OQS_KEM` in the first and second lines and re-run the templating?